app/templates/buyers/start_brief_info.html

```diff
@@ -40,10 +40,24 @@
     <div class="column-two-thirds large-paragraph">
         <div class="marketplace-paragraph">
             <p>
+                 {% set digital_specialists_intro %}
+                        You can use the Digital Marketplace to find a specialist, such as a developer or a user researcher, for a specific piece of work.
+                    </p> <p>
+                        You need to write 'requirements' to describe the work and tell suppliers to provide a specialist. The specialist can’t work for you outside the scope of your written requirements.
```

Comment:
These should be "fancy" typographical quotes around 'requirements'.

Reply:
Hi Kev
The style is for single quotes when we define something - see, for example, the first line on this page: https://www.gov.uk/tax-sell-home

Reply:
Yep single quotes - but they are curly, not straight.

Reply:
Thanks Kev. I've updated now those, if you wouldn't mind checking it again...